tests/test_fourier.py

```diff
@@ -71,12 +72,12 @@ def test_fourier_energy_conservation_2d():
 	np.random.seed(0)
 
 	for dtype in ['complex128', 'complex64']:
-		for shift_input in [[0,0],[0.1]]:
-			for scale in [1,2]:
+		for shift_input in [[0,0], [0.1]]:
+			for scale in [1, 2]:
 				for shift_output in [[0,0], [0.1]]:
-					for q in [1,3,4]:
-						for fov in [1,0.5,0.8]:
-							for dims in [[8,8],[8,16],[9,9],[9,18]]:
+					for q in [1, 1.23, 3, 4]:
+						for fov in [1, 0.5, 0.8]:
+							for dims in [[8,8], [8,16], [9,9], [9,18]]:
```

Comment:
Missing spaces in the list? [8,8] to [8, 8]?

Reply:
I tried to keep the pairs together, for visual reasons, but I'm fine with using spaces for all.

Reply:
In a couple lines up there are spaces in between the pairs. I wasn't sure which convention was chosen.

Reply:
I'll go full PEP8 then. Honestly I don't care that much for formatting of test code compared to library code, but I agree consistency is better. Same for below.

Reply:
Added spaces throughout the `test_optics.py` file.